packages/base/src/StaticAreaItem.js

```diff
@@ -59,7 +59,7 @@ class StaticAreaItem extends HTMLElement {
 			this._rendered = true;
 			updateShadowRoot(this.ownerElement, true);
 		}
-		await RenderScheduler.whenDOMUpdated(); // Wait for the content of the ui5-static-area-item to be rendered
+		await whenFinished(); // Wait for the content of the ui5-static-area-item to be rendered
```

Comment:
Without the RenderSchduler, whenFinished sounds too generic.

I also think the `when...` sounds good when followed by a `then...`

Maybe we can drop the `when` a use the `render` verb, something like
`await renderFinished()`